minijail/libminijail.c

```diff
@@ -3120,6 +3120,13 @@ static int minijail_run_internal(struct minijail *j,
 	state_out->child_env = NULL;
 	minijail_free_run_state(state_out);
 
+	/*
+	 * Set a parent death signal. The kernel will a SIGKILL to this process
```

Comment:
A verb is missing "The kernel will send a SIGKILL..."

Reply:
Thanks. done.